libs/components/src/next/button/stories/vl-button.stories-doc.mdx

```diff
@@ -97,6 +97,27 @@ Gebruik het `controlled` attribuut als je wil dat de button zich niet zelf aan-
 
 <Canvas of={VlButtonStories.ButtonToggle} />
 
+### CTA Link
+Gebruik een CTA link button om een call to action om een link af te beelden als een button.
```

Comment:
'Gebruik een CTA link button om een call to action om een link af te beelden als een button.'

Ik begrijp wat je wil zeggen en toch begrijp ik de zin niet.